products/spring-boot.md

```diff
@@ -1,92 +1,106 @@
-    support: 2022-11-24
+    extendedSupport: 2024-02-24
     latest: "2.6.14"
     latestReleaseDate: 2022-11-24
     releaseDate: 2021-11-19
+
 -   releaseCycle: "2.5"
     eol: 2022-05-19
-    support: 2022-05-19
+    extendedSupport: 2023-08-24
     latest: "2.5.14"
     latestReleaseDate: 2022-05-19
     releaseDate: 2021-05-20
+
 -   releaseCycle: "2.4"
     eol: 2021-11-18
-    support: 2021-11-18
+    extendedSupport: 2023-02-23
     latest: "2.4.13"
     latestReleaseDate: 2021-11-18
     releaseDate: 2020-11-12
+
 -   releaseCycle: "2.3"
     eol: 2021-05-20
-    support: 2021-05-20
+    extendedSupport: 2022-08-20
     latest: "2.3.12"
     latestReleaseDate: 2021-06-10
     releaseDate: 2020-05-15
+
 -   releaseCycle: "2.2"
     eol: 2020-10-16
-    support: 2020-10-16
+    extendedSupport: 2022-01-16
     latest: "2.2.13"
     latestReleaseDate: 2021-01-14
     releaseDate: 2019-10-16
+
 -   releaseCycle: "2.1"
     eol: 2019-10-30
-    support: 2019-10-30
+    extendedSupport: 2021-01-30
     latest: "2.1.18"
     latestReleaseDate: 2020-10-29
     releaseDate: 2018-10-30
+
 -   releaseCycle: "2.0"
     eol: 2019-03-01
-    support: 2019-03-01
+    extendedSupport: 2020-06-01
     latest: "2.0.9"
     latestReleaseDate: 2019-04-03
     releaseDate: 2018-03-01
+
 -   releaseCycle: "1.5"
     eol: 2019-08-06
-    support: 2019-08-06
+    extendedSupport: 2020-11-06
     latest: "1.5.22"
     latestReleaseDate: 2019-08-06
     releaseDate: 2017-01-30
-activeSupportColumn: true
-releaseColumn: true
-releaseDateColumn: true
-eolColumn: Security Support
-iconSlug: springboot
 
 ---
 
-> [Spring Boot](https://github.com/spring-projects/spring-boot) helps you to create Spring-powered, production-grade
-> applications and services with absolute minimum fuss. It takes an opinionated view of the Spring platform so that new
-> and existing users can quickly get to the bits they need.
+> [Spring Boot](https://github.com/spring-projects/spring-boot) helps you to create Spring-powered,
+> production-grade applications and services with absolute minimum fuss. It takes an opinionated
+> view of the Spring platform so that new and existing users can quickly get to the bits they need.
 
-See [Spring Boot Milestones page](https://github.com/spring-projects/spring-boot/milestones) for upcoming releases.
-See [Spring Boot Support page](https://spring.io/projects/spring-boot#support) for more details about support roadmap.
+See [Spring Boot Milestones page](https://github.com/spring-projects/spring-boot/milestones) for
+upcoming releases and [Spring Boot Support page](https://spring.io/projects/spring-boot#support) for
+more details about support roadmap.
 
 - Spring Boot 3.x requires **at least a Java 17 runtime**,
-- Spring Boot 2.7.5 (and therefore the latest Spring Framework 5.3.23) supports Java 19 while also remaining compatible with Java 11 and 8.
+- Spring Boot 2.7.5 (and therefore the latest Spring Framework 5.3.23) supports Java 19 while also
```

Comment:
` 2.7.5` -> ` 2.7.x`

Reply:
Does this apply all 2.7 releases or just `2.7.5+`?

Reply:
good question , let me check [ i have forgotten to look java19 support ]

Reply:
2.7.4 is the first one which supports java 19 so 2.7.4+ is more suitable { https://docs.spring.io/spring-boot/docs/2.7.4/reference/html/getting-started.html#getting-started }
also we need to mention that with 3.x users need Maven 3.5+  / Gradle 7.5 or later

Reply:
Same as https://github.com/endoflife-date/endoflife.date/pull/2377#discussion_r1087473982, this has to be fixed.

But that was not the goal of this PR (I only reformatted the description), so I filed a separate issue (for both Spring Framework and Spring Boot). See https://github.com/endoflife-date/endoflife.date/issues/2384.